scripts/save.rb

```diff
@@ -0,0 +1,2 @@
+
+\copy houses(name, motto, sigil) from '/westeros/houses.csv' csv delimiter as ',' header;
```

Comment:
This is the SQL way to import a csv file, but you're writing into a ruby script file